docs/pages/guides/using-sentry.mdx

```diff
@@ -204,6 +205,7 @@ Sentry.init({
     new Sentry.ReactNativeTracing({
       // Pass instrumentation to be used as `routingInstrumentation`
       routingInstrumentation,
+      enableNativeFramesTracking: Constants.appOwnership !== 'expo', // Only in native builds, not in Expo Go.
```

Comment:
We'll be deprecating the `appOwnership`. It's better to use `isRunningInExpoGo` function from the `expo` package 😉

Reply:
noice!